content/terraform-docs-common/docs/docs/tools/mcp-server/deploy.mdx

```diff
@@ -43,44 +53,164 @@ Refer to your client's documentation for instructions on enabling MCP:
 
 ## Install the server
 
-For VS Code, create an `mcp.json` file in the `.vscode` or `.cursor` directory of your local workspace and add the following configuration: 
+You can install the server using the following methods if you intend to run the server locally:
+
+- Install compiled the binary into your client. The binaries are available at [releases.hashicorp.com](https://releases.hashicorp.com/terraform-mcp-server/).
+- Install from source into your client.
+
+For remote installations, download the source to the remote and build the Docker image.
+
+### Compiled binary
+
+Refer to [Requirements](#requirements) for supported clients.
+
+<Tabs>
+
+<Tab heading="VS Code">
+
+To install the server so that it's available for all workspaces, add the following JSON block to your user setting JSON file:
 
 ```json
 {
-"servers": {
-   "terraform-mcp-server": {
+  "mcp": {
+    "servers": {
+      "terraform": {
+        "command": "docker",
+        "args": [
+          "run",
+          "-i",
+          "--rm",
+          "hashicorp/terraform-mcp-server"
+        ]
+      }
+    }
+  }
+}
+```
+
+Alternatively, you can install the server in specific workspaces. Create an `mcp.json` file in the `.vscode` directory of your workspace and add the following JSON block:
+
+```json
+{
+  "servers": {
+    "terraform": {
       "command": "docker",
       "args": [
-         "run",
-         "-i",
-         "--rm",
-         "hashicorp/terraform-mcp-server:0.1.0"
+        "run",
+        "-i",
+        "--rm",
+        "hashicorp/terraform-mcp-server"
       ]
-   }
+    }
+  }
 }
+```
+
+</Tab>
+
+<Tab heading="Claude Desktop, Amazon Q Developer and CLI">
+
+Create an `mcp.json` file and add the following configuration:
+ 
+```json
+{
+"mcpServers": {
+  "servers": {
+   "hcp-terraform": {
+     "command": "docker",
+     "args": [
+        "run",
+        "-i",
+        "--rm",
+        "hashicorp/terraform-mcp-server:0.1.0"
```

Comment:
Feel free to remove version from here too

Reply:
```suggestion
        "hashicorp/terraform-mcp-server:<version>"
```